index.bs

```diff
@@ -1660,6 +1334,94 @@ multiple transactions per microframe. In a SuperSpeed device this value will
 include the multiplication factor provided by the <code>bMaxBurst</code> field
 of the SuperSpeed Endpoint Companion descriptor.
 
+# Integrations # {#integrations}
+
+## Feature Policy ## {#feature-policy}
+
+This specification defines a <a>feature</a> that controls whether the
+{{Navigator/usb}} attribute is exposed on the {{Navigator}} object.
```

Comment:
Can we make `navigator.usb` print a warning to the console in addition to returning `undefined`?